### ISSUE TITLE
dialog.showMessageBox doesn't respect defaultId option on Windows

### ISSUE THREAD

**Issue by laurent22** [electron/electron#18258]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * v4.1.4
* **Operating System:**
  * Windows 10

### Expected Behavior

I've created some simple code to replicate the issue:

```
const { dialog } = require('electron');
const options = {
	type: 'question',
	message: 'Delete this note?',
	cancelId: 1,
	defaultId: 1,
	buttons: [ 'Delete', 'Cancel' ],
}
const result = dialog.showMessageBox(window, options);
console.info('This button was pressed: ' + result);
```

When the dialog shows up, the "Cancel" button should be selected by default since `defaultId` is set to 1. However, the default button is "Delete". This is confirmed by pressing Enter - in that case the message `This button was pressed: 0` is displayed.

![Capture](https://user-images.githubusercontent.com/1285584/57569134-5b20e480-73e8-11e9-8eb9-4adc238c7655.PNG)

### Actual Behavior

Instead of selecting Cancel by default it is selecting Delete, as if the `defaultId` option was ignored.

**Comment by MarshallOfSound** [comment]
This was fixed in #17547 and backport to `5-0-x`.  Can you upgrade to that version and test it out.  If the same issue occurs please @ mention me in a follow up comment below and I can re-open 👍

**Comment by Andrew-web-coder** [comment]
@MarshallOfSound I don't see any fixes in the latest version of Electron, it works as described by the original author
